kuma/wiki/views/document.py

```diff
@@ -872,9 +872,15 @@ def react_document(request, document_slug, document_locale):
     all_locales = (set([doc.locale]) |
                    set(trans.locale for trans in other_translations))
 
+    # Get the JSON data for this document
+    document_data = document_api_data(doc)
+    # And add another property to specify the requested locale
+    # (which may not be the same as the locale of the document)
+    document_data['requestLocale'] = document_locale
```

Comment:
I'm not advocating for the "_underscore prefix" but something/anything that sets it apart so that, when reading the code, you appreciate that some things come from state and some things come from runtime.

Reply:
Peter: that's an excellent point. Also, I've been talking with Ryan about handling redirects differently in kuma/api/v1/views.py, so that we just return the one redirect URL in that case and not a dict full of empty strings.

I propose to add a bit TODO comment for now, and merge this, but then go back in to handle both the redirect case and the request locale case so that the data structure we're using looks like this:

```
{
  documentData: { ... lots of stuff here, or just null for redirect ... }
  redirectURL: ... a string for redirect or null ...
  requestLocale: locale of the request as a string
}
```